app/views/personal_access_tokens/_personal_access_tokens_tab.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="tab-pane" id="personal_access_tokens">
+  <% #if authorized_for(hash_for_new_user_personal_access_token_path(:user_id => @user)) %>
+  <div id="personal_access_token_create"></div>
+  <%= mount_react_component('PersonalAccessToken', '#personal_access_token_create', {
+    'user_id': @user.id,
+    'csrf-token': form_authenticity_token
```

Comment:
imho no need to pass it per component, see https://github.com/theforeman/foreman/pull/4795/files#diff-35521d7d9795ca3a7f62a33bb3ceef6aR30